app/furniture/marketplace/tags/index.html.erb

```diff
@@ -1,16 +1,40 @@
 <%- breadcrumb :marketplace_tags, marketplace %>
 <%= render CardComponent.new do |card| %>
-  <%- marketplace.tags.each do |tag| %>
-    <div id="<%= dom_id(tag)%>" class="flex flex-row gap-3">
-      <span class="flex-grow">
-        <%= tag.label %>
-      </span>
+  <div class="flex flex-col gap-4">
+    <div>
+      <h1>Menu Groups</h1>
+      <%- if marketplace.tags.group_tag.empty? %>
```

Comment:
I like how in the "Tags" section there is a helpful message ("Adding tags to your products like "vegan" or "discounted" will help shoppers more easily find what they want"). Let's add something similar here, e.g: "Use product groups like "featured" or "produce" to display some of your products together, and to control the order in which products are displayed." (Or some better wording.)